docs/02-app/02-api-reference/05-next-config-js/trailingSlash.mdx

```diff
@@ -17,6 +17,8 @@ module.exports = {
 
 With this option set, urls like `/about` will redirect to `/about/`.
 
+For SSG sites, this option can be used to generate `/about/index.html` files instead of the default `/about.html` files.
```

Comment:
```suggestion
When used with [`output: "export"`](https://nextjs.org/docs/app/building-your-application/deploying/static-exports) configuration, the `/about` page will output `/about/index.html` (instead of the default `/about.html`).
```